src/argilla/server/models/mixins.py

```diff
@@ -50,6 +50,12 @@ def fill(self, **kwargs: Any) -> Self:
         for key, value in kwargs.items():
             if not hasattr(self, key):
                 raise AttributeError(f"Model `{self.__class__.__name__}` has no attribute `{key}`")
+            # If the value is a dict, set value for each key one by one, as we want to update only the keys that are in
+            # `value` and not override the whole dict.
+            if isinstance(value, dict):
+                dict_col = getattr(self, key, {})
```

Comment:
My bad. The previous line works when the attribute is not defined, but no when is defined with a None value
```suggestion
                dict_col = getattr(self, key) or {}
```